plugins/BEdita/Core/src/Filesystem/Thumbnail/GlideGenerator.php

```diff
@@ -54,10 +56,15 @@ class GlideGenerator extends ThumbnailGenerator
      * @param \BEdita\Core\Model\Entity\Stream $stream Stream entity instance.
      * @param array $options Thumbnail options.
      * @return string
+     * @throws BadRequestException
      */
     protected function getFilename(Stream $stream, array $options = [])
     {
         $ext = pathinfo($stream->file_name, PATHINFO_EXTENSION);
+        $fm = Hash::get($options, 'fm', 'jpg');
+        if (!in_array($fm, ['jpg', 'pjpg', 'png', 'gif', 'webp', 'avif'])) {
+            throw new BadRequestException(__d('bedita', 'Encoded the image to a specific format is not available.'));
```

Comment:
I'd include the invalid format in the exception message: it may be simpler to detect the misconfiguration:

```suggestion
            throw new BadRequestException(__d('bedita', 'Invalid thumbnail format: {0}', $fm));
```